examples/erc20Dividend.ts

```diff
@@ -117,53 +105,99 @@ window.addEventListener('load', async () => {
     name: ModuleName.ERC20DividendCheckpoint,
     address: erc20DividendAddress,
   });
-  const erc20Dividend = (await tickerSecurityTokenInstance.getModulesByName({
-    moduleName: ModuleName.ERC20DividendCheckpoint,
+
+  // Get General TM Address to whitelist transfers
+  const generalTMAddress = (await tickerSecurityTokenInstance.getModulesByName({
+    moduleName: ModuleName.GeneralTransferManager,
   }))[0];
+  const generalTM = await polymathAPI.moduleFactory.getModuleInstance({
+    name: ModuleName.GeneralTransferManager,
+    address: generalTMAddress,
+  });
+
+  // Add owner address in the whitelist to allow issue tokens
+  await generalTM.modifyKYCData({
```

Comment:
I think this is a very roundabout way of doing things. You should just KYC the investors directly and `issueMulti` to them, instead of KYCing the owner, issuing to him and then transferring from him to the investors

Reply:
Alright I think it should be ok now